core/server/services/xmlrpc.js

```diff
@@ -70,7 +67,14 @@ function ping(post) {
             timeout: 2 * 1000
         };
 
+        var goodResponse = /<member><name>flerror<\/name><value><boolean>0<\/boolean><\/value><\/member>/;
+
         request(pingHost.url, options)
+            .then(function (res) {
+                if (!goodResponse.test(res.body)) {
+                    throw new Error(res.body);
```

Comment:
Is it possible to parse out the error message? 🤔

Reply:
It is - but it's not nice, i'll add something, see what you think of it